feature/gribi/ate_tests/hierarchical_weight_resolution_test/hierarchical_weight_resolution_test.go

```diff
@@ -243,6 +247,23 @@ func (a *attributes) configSubinterfaceDUT(t *testing.T, intf *oc.Interface) {
 		} else {
 			s.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged().VlanId = ygot.Uint16(uint16(i))
 		}
+	}
+}
+
+func (a *attributes) configL3SubifDUT(t *testing.T, intf *oc.Interface) {
```

Comment:
Line 253-274 can be  merged to reduce code duplication

Reply:
Updated